packages/select/src/SelectItem.tsx

```diff
@@ -0,0 +1,44 @@
+import type { SharedSelectState } from './types';
+import type { ItemProps } from '@react-types/shared';
+import type { ComponentPropsWithoutRef, ElementType } from 'react';
+
```

Comment:
For anyone thoroughly confused by this -- `react-stately`'s collections use React components in a _slightly_ unique way. This is effectively a pass-through component that exposes `getCollectionNode` to be used internally by `react-stately` and `react-aria`.